free_fleet_examples/config/fleet/nav2_tb3_simulation_fleet_config.yaml

```diff
@@ -39,6 +39,8 @@ rmf_fleet:
       # For Nav2RobotAdapter
       navigation_stack: 2
       initial_map: "L1"
+      map_frame: "map"
+      base_footprint_frame: "base_link"
```

Comment:
we should use `robot_frame` since the base's footprint might not be the frame users want to use